src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -38,6 +40,13 @@ private Boolean HasMissingPackage
             }
         }
 
+        /// <summary>
+        /// Re-directs to a web link to get the feedback from the user. 
+        /// </summary>
+        private void ProvideFeedback(object sender, EventArgs e)
+        {
+            System.Diagnostics.Process.Start(FeedbackLink);
```

Comment:
this should be wrapped in a try

Reply:
Is this using whatever default explorer? Can you check the exception type which could be thrown from this call?

Reply:
``` 
Exceptions
InvalidOperationException
No file name was specified in the Process component's StartInfo. -or- The UseShellExecute member of the StartInfo property is true while RedirectStandardInput, RedirectStandardOutput, or RedirectStandardError is true.

Win32Exception
There was an error in opening the associated file.

ObjectDisposedException
The process object has already been disposed.

PlatformNotSupportedException
Method not supported on operating systems without shell support such as Nano Server (.NET Core only).
```

Reply:
Yes, added the try-catch here. Should all these exceptions be handled individually? Right now, I have a common exception message for all.